BlogPost.md

```diff
@@ -0,0 +1,297 @@
+# Deploying your application to StarlingX
+
+Deploying your own application to StarlingX is as easy as any other Kubernetes
+deployment out there. In this blog post we'll show you the process with a
+simple demonstration application.
+
+It's important to understand that an application can be deployed in many ways
+to the [Kubernetes cluster(s) that StarlingX manages](https://docs.starlingx.io/operations/k8s_cluster.html)
+:
+
+- [raw Kubernetes](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/);
+- [Helm](https://helm.sh/docs/intro/using_helm/#helm-install-installing-a-package);
+- [Flux](https://fluxcd.io/); and finally
+- StarlingX Application, which benefits from tight integration with the
+  [StarlingX system](https://opendev.org/starlingx/config).
+
+In this particular demonstration we will focus on [Helm](https://helm.sh/),
+which is the most popular package manager for Kubernetes. Future blog posts
+will address other deployment types and their advantages.
+
+Any developer that already packages their application with
+[Helm](https://helm.sh/) will be able to deploy their application in
+StarlingX without additional hassle.
+
+## The Demo App
+
+We will use a simple demonstration application for this blog post. We won't
+dive into the use case of the demonstration application, but there will be
+additional information about it at the end of the post.
+
+For now, let's focus on dealing with the deployment!
+
+## Deploying the Demo App
+
+### Via a Helm Package file
+
+> _TODO_:  this is a WIP section. Similar to https://docs.starlingx.io/deploy_install_guides/release/kubernetes_access.html#id3
+
+You can simply use the packaged Helm chart from your own source code, and
+use the [Helm CLI](needs-reference) to install the package.
+
+#### Package the Helm chart
+
+```shell
+git clone https://github.com/bmuniz-daitan/poc-starlingx-messages.git
+cd poc-starlingx-messages
+helm package helm-chart/
+```
+
+The above commands should create a `poc-starlingx-<version>.tgz` file.
+
+```shell
+...
+Successfully packaged chart and saved it to: .../poc-starlingx-messages/poc-starlingx-1.5.2.tgz
+```
+
+#### Install the Helm package directly on StarlingX
+
+Once you made the package available on a StarlingX instance, the below command
+will deploy the application to the StarlingX-managed Kubernetes cluster:
+
+```shell
+sysadmin@controller-0:~$ source /etc/platform/openrc; helm install poc-starlingx-messages-node poc-starlingx-1.5.2.tgz
```

Comment:
Do the user need admin credentials in order to run helm commands?

Reply:
Not really, no. Removing.